packages/libs/react-ui/src/components/Notification/NotificationContainer.tsx

```diff
@@ -0,0 +1,71 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  cardTitleClass,
+  colorVariants,
+  containerClass,
+  containerClassRightPadded,
+  contentClass,
+  expandVariants,
+  iconContainerClass,
+  iconContainerExpandedClass,
+} from './Notification.css';
+import { NotificationButton } from './NotificationButton';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcon)[keyof typeof SystemIcon];
+  title?: string;
+  children?: React.ReactNode;
+  expanded?: boolean;
+  color?: keyof typeof colorVariants;
+  hasCloseButton?: boolean;
+  onClose?: () => void;
+}
+
+export const NotificationContainer: FC<INotificationProps> = ({
+  icon,
+  title,
+  children,
+  hasCloseButton = false,
+  color = 'primary',
+  expanded = false,
+  onClose,
+}) => {
+  const Icon = icon || SystemIcon.HelpCircle;
+
+  const classList = classNames(
+    hasCloseButton ? containerClass : containerClassRightPadded,
+    colorVariants[color],
+    expandVariants[expanded ? 'true' : 'false'],
+  );
+
+  return (
+    <div className={classList}>
+      <span className={iconContainerClass}>
+        <Icon size={'md'} />
+      </span>
+
+      <div>
+        <p className={contentClass}>
+          {title !== undefined && <h4 className={cardTitleClass}>{title}</h4>}
+          <p>{children}</p>
+        </p>
+      </div>
+
+      <span
+        className={expanded ? iconContainerExpandedClass : iconContainerClass}
+      >
+        {hasCloseButton && (
+          <NotificationButton
+            onClick={onClose}
+            icon={SystemIcon.Close}
+            color={color}
+          ></NotificationButton>
+        )}
+      </span>
```

Comment:
Given the way that this is written, there can be an empty span. Why don't you create a custom closeButtonClass and just render the button using the ternary?